docs/_guide/lifecycle.md

```diff
@@ -47,6 +47,22 @@ LitElement also inherits the default [lifecycle callbacks](https://developer.moz
 
 </div>
```

Comment:
This shouldn't be inside an alert box. We try to avoid putting two boxes in a row, and in this case it's introducing an example, which _isn't_ in the alert box.

Reply:
Okay, got it.
Thank you